docker/Dockerfile

```diff
@@ -6,26 +6,19 @@
 # BUILD: docker build --network=host --file docker/Dockerfile --tag pynvml .
 # RUN: docker run --gpus all --rm -it  pynvml
 ##############################################################
-FROM python:3.6
+FROM python:3.8
 
 ENV DEBIAN_FRONTEND noninteractive 
 
-ADD ./requirements.txt /app/requirements.txt
-
-RUN pip install --no-cache-dir --upgrade pip && \
-    pip install --no-cache-dir -r /app/requirements.txt && \
+ADD . /app/.
+RUN cd /app && \
+    pip install --no-cache-dir --upgrade pip && \
+    pip install --no-cache-dir . && \
     rm -rf ~/.cache/pip
 
-ADD ./pynvml /app/pynvml
-ADD ./setup.py /app/
-ADD ./setup.cfg /app/
-ADD ./README.md /app/
-ADD ./PKG-INFO /app/
-ADD ./MANIFEST.in /app/
-ADD ./help_query_gpu.txt /app/
 ADD docker/entrypoint.sh /docker-entrypoint.sh
 
-RUN chmod +x  /docker-entrypoint.sh 
+RUN chmod +x /docker-entrypoint.sh 
```

Comment:
Given the changes above, we can drop this entrypoint as those steps are subsumed by...

1. Doing an editable install `-e` with `pip`
2. Cleaning up Python bytecode files/directories